velox/functions/prestosql/tests/MapTest.cpp

```diff
@@ -25,7 +25,64 @@ using namespace facebook::velox::functions::test;
 
 namespace {
 
-class MapTest : public FunctionBaseTest {};
+class MapTest : public FunctionBaseTest {
+ public:
+  template <typename T>
+  void testFloatingPointCornerCases() {
+    static const T kNaN = std::numeric_limits<T>::quiet_NaN();
```

Comment:
nit: just reordering test cases (i.e creation of vector and then test case for it) will help out in readability.